lib/usbd/backend/usbd_dwc_otg.c

```diff
@@ -109,12 +109,25 @@ static inline uint16_t get_fifo_depth(usbd_device *dev)
  */
 static void flush_fifo(usbd_device *dev)
 {
+	// Set Global IN OUT ep NACK
+	REBASE(DWC_OTG_DCTL) = DWC_OTG_DCTL_SGONAK | DWC_OTG_DCTL_SGINAK;
+
+	/* Wait for AHB idle. */
+	while (!(REBASE(DWC_OTG_GRSTCTL) & DWC_OTG_GRSTCTL_AHBIDL));
+
+	const uint32_t GNACKEFF = DWC_OTG_GINTMSK_GINAKEFFM ; // | DWC_OTG_GINTMSK_GONAKEFFM ;
+	// Wait Global NACK Effective
+	while ( (REBASE(DWC_OTG_GINTSTS) & GNACKEFF) != GNACKEFF );
```

Comment:
`while ((REBASE(DWC_OTG_GINTSTS) & GNACKEFF) != GNACKEFF);`  extra space.
codebase i saw didn't had any such space after bracket open or close. we can just stick to it.